salt/orchestrate/edx/backup.sls

```diff
@@ -71,7 +90,7 @@ mount_backup_drive:
         - mount.mounted
     - kwarg:
         name: /backups
-        device: /dev/xvdb
+        device: /dev/{{ salt.grains.get('ec2:block_device_mapping:ebs2') }}
```

Comment:
If we use `ec2.create_attach_volume` then we can just use the same device name as what is specified in the call to that action.